.github/workflows/main.yml

```diff
@@ -0,0 +1,38 @@
+name: CI
+
+on: [push, pull_request]
+
+env:
+  COLORTERM: 'yes'
+  TERM: 'xterm-256color'
+  PYTEST_ADDOPTS: '--color=yes'
+
+jobs:
+  ansible-lint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+      - name: Install ansible-lint
+        run: pip install ansible-lint
+      - name: Run tests
+        run: |
+          ansible-lint -r rules/ playbooks/*
+          ansible-lint -r rules/ roles/*
```

Comment:
Have you considered using https://github.com/marketplace/actions/ansible-lint

Reply:
Not yet, no

Reply:
Now I did, and it uses an older version of ansible-lint and meh.

Reply:
I also tried it in the past and didn't really saw a benefit. At least last time it always needed to build the container so it wasn't faster.